source/documentation/breaking-changes/extend-compound.liquid

```diff
@@ -0,0 +1,114 @@
+---
+title: "Breaking Change: Extending Compound Selectors"
+introduction: >
+  LibSass currently allows compound selectors like `.message.info` to be
+  [extended](/documentation/at-rules/extend), but the way it was extended doesn't match the
+  way `@extend` is meant to work.
+---
+
+{% markdown %}
+{% compatibility true, false, null, false %}{% endcompatibility %}
+
+When one selector extends another, Sass styles all elements that match the
+extender as though they also match the class being extended. In other words, if
+you write `.heads-up {@extend .info}`, it works just like you replaced
+`class="heads-up"` in your HTML with `class="heads-up info"`.
+
+Following that logic, you'd expect that `.heads-up {@extend .message.info}` to
+work like replacing `class="heads-up"` with `class="heads-up info message"`. But
+that's not how it works right now in LibSass and Ruby Sass--instead of adding
+`.heads-up` to every selector that has *either `.info` or `.message`*, it only
+adds it to selectors that have *`.info.message` together*.
+{% endmarkdown %}
+
+{% codeExample 1, false %}
```

Comment:
Not specific to this file, but it would be cool if the `codeExample` tag could be nested to more closely follow other liquid tags. Right now it seems to be confused if the `===` separator has leading whitespace. Also, if the `exampleName` is not provided we could generate the indeces ourselves, to avoid having to pass them for compatibility with the old site.

Reply:
Earlier we discussed auto-generating, and our eventual conclusion was that requiring an explicit name might be preferred so that URLs are consistent. That is, we wanted to avoid the case where adding a new example at the top of a page changes the URLs for all the other later examples on the page.